### ISSUE TITLE
Google Contacts not working (anymore?)

### ISSUE THREAD

**Issue by code-chicken** [jitsi/jitsi#444]
A few days ago I installed Jitsi and I am having the same problems as already explained in this issue: #323 . However, when I read the articles it looks like it was solved and after that the discussion was about Fritz!Box. So I decided to create a new issue.

As with the old issue: With each letter that I enter there pops up a window and asks me for the auth code from google. I could successfully retrieve the authorization code from Google but it seems that the code is not being saved. This also happens if I run Jitsi as an Administrator.

This is my environment:
Jitsi 2.10.5550
Java 1.8.0_151
Windows 10 (1709 with all updates applied)

![image](https://user-images.githubusercontent.com/3024019/36095191-73a6d6f6-0ff1-11e8-90c4-8e4dfda4f0a5.png)

**Comment by Leyline77** [comment]
+1, I have had the same issue, exactly as described, for many versions now too.
Can we please re-open this issue, or have it explained if it it currently not solvable anymore.

**Comment by ibauersachs** [comment]
I've tried to fix some code and show an error when the entered code in the dialog is invalid. In general however, this should still work as is.

Note that the code shown on the Google sign-in page can only be used once, so any attempt to reuse it will (currently) fail silently.

**Comment by ibauersachs** [comment]
Well, nightly is a bit exaggerated. A new build become available as soon as something is committed. There wasn't a commit since December since the certificate for Windows has expired and we need to solve an issue before we can enable the builds again.

I don't think there are any changes between 2.10 and the December build, it was most likely an expired refresh token that you solved by revoking the access on the accounts page. Thanks for the heads-up!

**Comment by bsilvern** [comment]
Just want to say I had exactly the same issue as code-chicken. Added and removed Jitsi permission in my Google account perhaps 8 times before arriving here and then trying the nightly build as code-chicken mentioned. Problem solved!
